chibuezeayogu/blog_portfolio
# Devise authentication

**PR Description**
- The PR Implements `Devise Authentication`

**This change addresses the need by:** 
 - Implemented Devise Base Authentication along with User model
 - Added Virtual Attributes 


**How should this be manually tested?**
* git pull and checkout into `chibueze/authentication`
* run this using the command `rails start`,  start the application
* Ensure you can perform the actions on `This change addresses the need by`

**Relevant Trello links (if applicable)**
* N/A
